src/injected/components/details-dialog.tsx

```diff
@@ -186,9 +186,11 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
         const ruleNameID = 'rule-name';
 
         return (
-            <div className="insights-dialog-rule-name" aria-labelledby={ruleNameID}>
-                {this.renderSectionTitle('Rule name', ruleNameID)}
-                <NewTabLink href={fixUrl(rule.helpUrl)}>{rule.ruleId}</NewTabLink>
+            <div className="insights-dialog-rule-name">
```

Comment:
did you try changing the original `<div>` to be a `<section>` instead of adding a new `<section>` tag as direct child of the origin `<div>`... mostly curious about the exploration process than anything.

Reply:
I did not try changing the original `<div>` to be a `<section>` tag; if I remember correctly, the example that we looked at to come up with this fix had the section tag nested within a div, and we were trying to keep the same structure. That said, I can try making the parent div a section tag and add the aria-labelledBy attribute back? Or are you ok with keeping it like this?

Reply:
I think less "extra" tags in the general html structure is better, if we can go with changing the original `<div>` to `<section>`, that would be better.

Reply:
Note: updated to follow @smoralesd 's suggestion; converted `<div>` to a `<section>` tag and have both the className and aria-labelledBy as attributes for the `<section>` tag